pkg/api/api.swagger.go

```diff
@@ -9,7 +9,7 @@ package api
 // SwaggerJsonTemplate is a generated function returning the template as a string.
 // That string should be parsed by the functions of the golang's template package.
 func SwaggerJsonTemplate() string {
-	var tmpl = "{\n" +
+	tmpl := "{\n" +
```

Comment:
You'll have to revert this change (and the analogous one below), or you'll break the tests; see #3084.